plutus-metatheory/src/Untyped/Equality.lagda.md

```diff
@@ -251,21 +262,24 @@ decEq-⟦ _⊢♯.list t ⟧tag (x U.∷ v) (x₁ U.∷ v₁) with decEq-⟦ t 
 ... | yes refl with decEq-⟦ _⊢♯.list t ⟧tag v v₁
 ...                  | yes refl = yes refl
 ...                  | no ¬v=v₁ = no λ { refl → ¬v=v₁ refl }
-decEq-⟦ _⊢♯.pair t t₁ ⟧tag (proj₁ U., proj₂) (proj₃ U., proj₄) with (decEq-⟦ t ⟧tag proj₁ proj₃) ×-dec (decEq-⟦ t₁ ⟧tag proj₂ proj₄)
+decEq-⟦ _⊢♯.array t ⟧tag = decEq-Array-TyTag {t} decEq-⟦ t ⟧tag
```

Comment:
Since `decEq-Array-TyTag` is postulated and doesn't have a Haskell definition, won't this crash at runtime?